### ISSUE TITLE
Don't work with Fixtures?

### ISSUE THREAD

**Issue by michaelmeneses** [mtarld/symbok-bundle#49]
Hi,

I am confused if symbok-bundle does work with Fixtures classes or I did something wrong?

**Comment by mtarld** [comment]
Hi!

What kind of fixtures are you talking about? Can you parse a tiny example?

**Comment by michaelmeneses** [comment]
I add Data annotation in my class and run `php bin/console symbok:update:classes`.
My class was updated: Did add @method annotation. Ok!

But I use bin/console to execute Doctrine Fixtures don't work.

```
php bin/console doctrine:fixtures:load -n

   > purging database
   > loading App\DataFixtures\AvaFixtures
   > loading App\DataFixtures\ProgramaFixtures

In ProgramaFixtures.php line 24:
                                                                                         
Attempted to call an undefined method named "setNome" of class "App\Entity\Programa".
                                                                                         

doctrine:fixtures:load [--append] [--group GROUP] [--em EM] [--shard SHARD]
[--purge-with-truncate] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version]
[--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
```


My Class:

```
<?php

namespace App\Entity;

use ApiPlatform\Core\Annotation\ApiResource;
use Doctrine\ORM\Mapping as ORM;
use Mtarld\SymbokBundle\Annotation\Data;

/**
 * @ApiResource ()
 * @Data ()
 * @ORM\Entity (repositoryClass=ProgramaRepository::class)
 * @method mixed __construct(int $id, string $nome)
 * @method int|null getId()
 * @method self setId(int $id)
 * @method string|null getNome()
 * @method self setNome(string $nome)
 */
class Programa { (...)
```

**Comment by mtarld** [comment]
Thanks for the example, I'll have a look ASAP, it's strange indeed

**Comment by michaelmeneses** [comment]
I am using Symfony 5.0.8

**Comment by mtarld** [comment]
Indeed I succeeded to reproduce it.
It seems to be related to the class loading order in Symfony console context. Therefore every command is affected.

Thanks for the report!

**Comment by mtarld** [comment]
I was wrong about my previous guess. 

As mentioned in #50, only some of doctrine related commands were affected. You can have a look at the detailed description in the related PR, which should fix it in 2.2.0
